CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2.0.0a0
+
+- Support Python 3.9
+- Require polliwog >= 3.0.0a0 and lacecore == 3.0.0a1
```

Comment:
Lacecore is a dev dependency so doesn't need to be here

Reply:
fixed.